src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -22,6 +23,15 @@
 logger = get_logger(__name__)
 
 # pylint: disable=line-too-long
+# pylint: disable=bare-except
+
+
+def cleanup_helm_chart_directory(path):
+    try:
+        if os.path.isdir(path):
+            shutil.rmtree(path)
+    except:
```

Comment:
>except [](start = 4, length = 6)

What are the chances of this happening?

Reply:
This might happen if some other program(not terminal) is accessing the charts directory. I saw this when I had opened the charts in VS code. But same applies to .azure directory which is anyways reqd by az cli to fetch cached tokens.